README.md

```diff
@@ -145,9 +145,10 @@ test result: ok. 2 passed; 0 failed; 0 ignored; 0 measured; 2 filtered out
 
 ## Integration with Cargo.
 
-Tests created with lang_tester can be used as part of an existing test suite.
-The `Cargo.toml` file can be modified so that tests created with lang_tester
-will run with the `cargo test` command.
+Tests created with lang_tester can be used as part of an existing test suite and
+can be run with the `cargo test` command. For example, if the Rust source file
+that runs your lang tests is `lang_tests/run.rs` then add the following to your
+Cargo.toml
```

Comment:
Add a `:` to the end of this sentence and we're golden!

Reply:
Done !